db/migrate/20220830093508_set_duration_for_work_packages_having_dates.rb

```diff
@@ -0,0 +1,10 @@
+class SetDurationForWorkPackagesHavingDates < ActiveRecord::Migration[7.0]
+  def self.up
+    execute <<~SQL.squish
+      UPDATE work_packages SET duration = due_date - start_date + 1
+        WHERE due_date IS NOT NULL AND start_date IS NOT NULL;
+      UPDATE work_package_journals SET duration = due_date - start_date + 1
+        WHERE due_date IS NOT NULL AND start_date IS NOT NULL;
+    SQL
```

Comment:
The [original migration set the value to 1](https://github.com/opf/openproject/blob/dev/db/migrate/20220525154549_add_duration_to_work_packages.rb#L17-L28) if start or due date was missing. Those work packages' durations should be set to NULL.

Reply:
Sorry, only now saw the other migration up above, that does exactly that.

Reply:
@ulferts I consolidated the two migrations into one as it makes more sense. I also removed the condition to apply the change when the feature flag is enabled.
8078f5bc62fb3f80602e0859f11e4bb6323ca6ec